omero/sysadmins/omeroweb-upgrade.rst

```diff
@@ -87,6 +87,48 @@ All official OMERO.web plugins can be installed from PyPI_.
 You should remove all previously installed plugins and install the latest
 versions using pip.
 
+.. _clearing_session_store:
+
+Clear the sessions store (optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+During an upgrade, it might be necessary to clear the sessions store
+before restarting OMERO.web. This is the case when either OMERO.web or
+Django introduced backwards incompatible changes in the way sessions are
+stored or if the session serialization format has been modified via
+:property:`omero.web.session_serializer`.
+
+.. warning::
+
+   Clearing the sessions store will terminate all OMERO.web sessions and log
+   out all users.
+
+The process for clearing the store depends on the storage backend configured
+via :property:`omero.web.session_engine`:
+
+- if :property:`omero.web.session_engine` is either unset or set to
+  `omeroweb.filesessionstore` or `django.contrib.sessions.backends.file`,
+  OMERO.web will use file-based sessions. The session files are stored under a
+  temporary folder determined by the output of `tempfile.gettempdir()`,
+  usually `/tmp` and prefixed either by the value of
+  :property:`omero.web.session_cookie_name` if the property is set or by
+  `sessionid` if the property is unset.
+  For a default configuration, the following command will
```

Comment:
maybe indicate earlier that by default it is unset
something like ``` :property:`omero.web.session_engine` (default: unset)```

Reply:
This will be true also if `omero.web.session_engine` is explicitly set to a file backend.

By default configuration, I was rather thinking to the expectation that `SESSION_COOKIE_NAME` and possibly other Django envvars would not be overriden. Maybe I'll invert the two sentence here i.e. first give the command that should work in 95% of the cases then mention the variables that could modify the location of the temporary sessions

Reply:
The way it is written at the moment made me think that the default was unset.
Swapping the order should avoid the confusion